### DISCUSSION TITLE
Heltec Sufficient IoT Hub transport node...

### DISCUSSION THREAD

**Discussion by cobraPA** [markqvist/Reticulum#510] [General]
Anyone looking for low cost, compact, low-medium power transport nodes?  I'm experimenting with this new ARM based board that has a Debian image.  It's a fairly new device from Heltec, but the bonus is on board LoRa radio (SX1262), wifi and ethernet.  They also have a cell modem add on board, although no docs I've seen for that.

**Comment by markqvist** [comment]
That's pretty cool. If you take a shot at it, let us all know how easy/hard it is to get RNS up and running on it!

**Reply by cobraPA** [reply]
Haha, well, I got RNS running today!  I'd call it medium.  I worked through getting a newer python version running and had to use rnspure because of the usual challenges with python cryptography.  The image Heltec supplies is Debian buster for ARM, and it's kind of slow... so building things on the target took a while.  I was able to get rnstatus to show a SidebandOS peer on wifi so far, so first basic test.  I really want to get it to the point of a LoRa transport node, so that needs a local interface for LoRa.  I expect wifi and ethernet will just work, but will do some testing there.
I have a couple more on order, so if you want one @markqvist, I can donate to you.
I also reached out to Heltec about the img they supply.  They don't give much info about how it is generated and their changes, so hoping they will share a little more.

**Reply by markqvist** [reply]
That's awesome to hear. It really looks like there's a lot of potential for a board like this, and I'm quite intrigued by it as well. That's really kind of you, and I'd love to have one, but realistically, I won't have time to look at it for the next two months or so, so it's much better if you keep them and work on them yourself :)

I've been wanting to create a bare-minimum and efficient OS image for stuff like this for a while, and a board like this would be very good to use as the initial development target! I'll probably get a couple of them later in the year to work on :)

**Reply by cobraPA** [reply]
Sounds good.  Yeah, they take a while to get from the Heltec store, with shipping out of China.  About a month or so to the USA so far.
